xrviz/tests/test_example.py

```diff
@@ -0,0 +1,7 @@
+from .. import example
+from xrviz.dashboard import Dashboard
+
+
+def test_dashboard_example():
+    dash = example(False)
```

Comment:
I have only checked case where `show=False`. `show = True` directly opens in the browser and has type `None`.

Reply:
(`show=True` blocks and doesn't return at all. We could get fancy and do a times interrupt signal to test, but let's not).